build/luarocks/BUILD.luarocks.bazel

```diff
@@ -39,8 +39,8 @@ configure_make(
 kong_template_genrule(
     name = "luarocks_exec",
     srcs = [
-        "@openssl//:openssl",
-        "@libexpat//:libexpat",
+        "@libexpat",
```

Comment:
Although it can be written this way, it's better to stick with the previous approach to avoid confusion for those who aren't familiar with Bazel.